stress_test_model_eq.R

```diff
@@ -165,7 +168,12 @@ pacta_equity_results <- read_pacta_results(
     scenario_geography_filter = scenario_geography_filter,
     scenarios_filter = scenarios_filter,
     equity_market_filter = cfg$Lists$Equity.Market.List
-  )
+  ) %>%
+  dplyr::group_by(company_name) %>%
+  mutate(
+    term = round(runif(n = 1, min = 1, max = 10), 0) # TODO: temporary addition, needs to come directly from input
+  ) %>%
+  ungroup()
```

Comment:
could you please declare missing namesspaces? I have already opened a PR that did the rampage for all  other existing code